packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -1629,6 +1635,14 @@ class _RenderScrollSemantics extends RenderProxyBox {
     markNeedsSemanticsUpdate();
   }
 
+  void _onScrollToOffset(Offset targetOffset) {
+    final double offset = switch (axis) {
+      Axis.horizontal => targetOffset.dx,
```

Comment:
will there be cases where dx and dy both none zero?

Reply:
Not currently. In the iOS implementation, every semantics node is only scrollable in one axis I think, so iOS won't issue commands that would scroll both axes. If it does then I think we can't do anything other than just ignoring the unscrollable axis?

Reply:
oh ok, just want to double check there isn't any surprise.